core/domain/classifier_services.py

```diff
@@ -124,6 +124,57 @@ def classify_string_classifier_rule(state, normalized_answer):
 
     return None
 
+def train(exploration):
```

Comment:
OK, so the way I see it is:
- I'm still kind of confused about the benefit of merging this PR. If it depends on other things being implemented then why don't we implement those things first?
- On the other hand, the main thing I worry about when thinking about whether to merge something or not is whether it will break existing code. We aren't ready to turn on a classifier yet, I believe, so I don't understand what you mean by "get the pipeline flowing" -- in fact, that sounds kind of scary if we're planning to deprecate the LDA classifier.

Bearing these thoughts in mind, what should we do? If you still think that this should be merged then I would like to understand (i) why it's important to do so now and not later, (ii) what side-effects this PR will have.